src/index.html

```diff
@@ -1,16 +1,51 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8"
+    >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"
+    >
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form big-form" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      data-qa="big">
+      <label 
+        for="searchbar" 
+        class="form__lable">
+      </label>
+      <input 
+        type="text" 
+        name="searchbar" 
+        id="searchbar" 
+        class="form__big-input input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      data-qa="small"
+      class="form small-form">
+      <label 
+        for="searchbar2" 
+        class="form__lable">
+      </label>
+      <input type="text" 
+        class="form__small-input input" 
+        name="searchbar" 
+        id="searchbar2"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
Now it's hard to read your HTML code. Please pay attention to formatting HTML code.
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 5th point